pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -1596,6 +1596,36 @@ void createSubscription(String topic, String subscriptionName, MessageId message
     Map<BacklogQuota.BacklogQuotaType, BacklogQuota> getBacklogQuotaMap(String topic, boolean applied)
             throws PulsarAdminException;
 
+    /**
+     * Get backlog size by its messageId.
```

Comment:
What is backlog here? The number of messages or the size in bytes?

Reply:
yes, it's backlog size int bytes